### DISCUSSION TITLE
DRG In-Network Rates - What negotiation arrangement value is being used?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#125] [Q&A]
Since the DRGs involve standard grouper processing based on the version, could we us the 'ffs' negotiated arrangement and report the pricing based on each applicable DRG code for that version in the In-network Rate File?

If DRGs are reported as bundle, then what is expected in the bundle coverage array?  Hopefully, it is not expected to report the ICDs defined for a given DRG code--as many of the same procedures are valid for other DRGs and they are defined in the grouper for the version. 

What are others planning for the negotiated arrangement value for MS-DRG  and APR-DRG?

**Comment by reevesj2** [comment]
It's a great question.   Originally the plan was just to report the DRG and the associated payment value for that DRG.  I didn't realize there was a bundle section until recently (late to the game) so I don't know if that still makes sense.

**Reply by tdfow** [reply]
We are currently planning to use the FFS negotiated arrangement with the negotiated type as 'derived' to identify the derived rate for each DRG code for the version(s) of DRG that need to be supported within the in-network rate file.

**Reply by reevesj2** [reply]
I was leaning towards using derived for the Negotiated Type, but was just reading the definitions in the schema.  

**negotiated:** If applicable, the negotiated rate, reflected as a dollar amount, for each covered item or service under the plan or coverage that the plan or issuer has contractually agreed to pay an in-network provider, except for prescription drugs that are subject to a fee-for-service reimbursement arrangement, which must be reported in the prescription drug machine-readable file. If the negotiated rate is subject to change based upon participant, beneficiary, or enrollee-specific characteristics, these dollar amounts should be reflected as the base negotiated rate applicable to the item or service prior to adjustments for participant, beneficiary, or enrollee-specific characteristics.

**derived:** If applicable, the price that a plan or issuer assigns to an item or service for the purpose of internal accounting, reconciliation with providers or submitting data in accordance with the requirements of 45 CFR 153.710(c).

I'm not sure Derived makes sense for DRG.  Our DRG's have a set rate used for payment, while derived makes it sound more like it's an amount used more for recordkeeping purposes.  I'm leaning more towards Negotiated.  Thoughts?

**Reply by bryanbriegel** [reply]
One thing to consider re supporting using negotiated vs derived - is consider 45 CFR 153.710(c) where issuers may have a capitated arrangement and do not have claim financials to submit for CMS's RARI programs, they are compelled to derive a financial for those claims when submitting to EDGE servers.  

They have to price those encounters: "..that does not generate individual enrollee claims in the normal course of business must derive the costs of all applicable provider encounters using its principal internal methodology for pricing those encounters. If the issuer does not have such a methodology, or has an incomplete methodology, it must supplement the methodology in a manner that yields derived claims that are reasonable in light of the specific service and insurance market that the plan is serving."   It sounds like you have set 'negotiated' rates.

**Reply by nmayle** [reply]
@bryanbriegel seems to be confirming that "derived" is meant to only be used with a negotiation_arrangement of "capitation" and otherwise the contractual rated should be the value in negotiated_price.

@shaselton is that the official answer?  We have seen files using "derived" in other cases, so it may not be clear to the file creators.
